plugins/sourcegraph/src.go

```diff
@@ -0,0 +1,22 @@
+package sourcegraph
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func SourcegraphCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Sourcegraph CLI",
+		Runs:      []string{"src"},
+		DocsURL:   sdk.URL("https://docs.sourcegraph.com/cli"),
+		NeedsAuth: needsauth.NotForHelpOrVersion(),
```

Comment:
Should we opt out of authentication in the case where no arguments are provided, as well?